iothub/device/src/RecommendedAction.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Client
+{
+    /// <summary>
+    /// Recommended actions for users to take upon different ConnectionStatus and ConnectionStatusChangeReason
+    /// </summary>
+    public enum RecommendedAction
```

Comment:
I think the default value (first enum) should be what the state the SDK is in before OpenAsync is called.

Reply:
I don't have a better name of the default one, so keep it as-is. Open to suggestions. :)